<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StrTelive</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBox_TD">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start TETRAD</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Telive</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_GNU">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>85</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start GNU</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select number of Telive Receivers</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>361</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Receiver1</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox1">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_rcv1">
     <property name="text">
      <string>UDP Port</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_FR1">
     <property name="intValue" stdset="0">
      <number>42000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  -</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_TO1"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>361</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Receiver2</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox2">
     <property name="maximum">
      <number>6</number>
     </property>
     <property name="value">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_rcv2">
     <property name="text">
      <string>UDP Port</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_FR2"/>
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  -</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_TO2"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>120</y>
      <width>361</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Receiver3</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox3">
     <property name="maximum">
      <number>6</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_rcv3">
     <property name="text">
      <string>UDP Port</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_FR3"/>
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  -</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_TO3"/>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>160</y>
      <width>361</width>
      <height>27</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Receiver4</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox4">
     <property name="maximum">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_rcv4">
     <property name="text">
      <string>UDP Port</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_FR4"/>
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  -</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_TO4"/>
   </widget>
   <widget class="QLabel" name="MyMsg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_FIFO">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Use FIFO File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_UDP">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Use UDP</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Settings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_Settings">
   <property name="text">
    <string>Save Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
